index.js

```diff
@@ -55,8 +55,8 @@ class ServerlessCustomDomain {
 
     const createDomainName = this.getCertArn().then(data => this.createDomainName(data));
     return Promise.all([createDomainName])
```

Comment:
You can remove Promise.all, it's redundant here because you only work with 1 promise

Reply:
Ah, nice catch. Not something we changed in this PR but I'll fix it anyway